packages/terra-date-picker/src/react-datepicker/month.jsx

```diff
@@ -196,10 +207,18 @@ export default class Month extends React.Component {
   render() {
     const getClassNames = cx({
       'react-datepicker-month': true,
+      'react-datepicker-body': true,
       'react-datepicker-month--selecting-range': this.props.selectingDate && (this.props.selectsStart || this.props.selectsEnd)
     });
     return (
-      <div tabIndex="0" className={getClassNames} onMouseLeave={this.handleMouseLeave} role="listbox">
+      <div
+        tabIndex="0"
+        className={getClassNames}
+        onMouseLeave={this.handleMouseLeave}
+        role="application"
+        aria-label={`${utils.getLocalizedDateForScreenReader(this.props.preSelection, { intl: this.props.intl, locale: this.props.locale} )}. To change the selection, use the arrow keys. Press Enter to select a date. Press Escape to close date picker popup.`}
```

Comment:
Todo: Get translation

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/866/commits/6ea023ec6abb090fc1b5d30e23041f9ddbddad13